shinylive/_main.py

```diff
@@ -87,151 +129,315 @@ def export(
     )
 
 
-@main.command(
-    help=f"""Manage local copy of Shinylive web assets
-
-    \b
-    shinylive Python package version: {_version.SHINYLIVE_PACKAGE_VERSION}
-    shinylive web assets version:     {_assets.SHINYLIVE_ASSETS_VERSION}
+# #############################################################################
+# ## Assets
+# #############################################################################
 
-    \b
-    Commands:
-      download: Download assets from the remote server.
-      cleanup: Remove all versions of local assets except the currently-used version, {_assets.SHINYLIVE_ASSETS_VERSION}.
-      remove: Remove a specific version of local copies of assets. Must be used with --version.
-      info: Print information about the local assets.
-      install-from-local: Install shinylive assets from a local directory. Must be used with --source.
-      link-from-local: Create a symlink to shinylive assets from a local shinylive build directory. Must be used with --source. This is useful when doing development on the Shinylive web assets.
 
+@main.group(
+    help=f"""Manage local copy of Shinylive web assets
+    {version_txt}
 """,
     no_args_is_help=True,
 )
-@click.argument("command", type=str)
+def assets() -> None:
+    pass
+
+
+def upgrade_dir(dir: str | Path | None) -> Path:
+    if dir is None:
+        dir = _assets.shinylive_cache_dir()
+    dir = Path(dir)
+    return dir
+
+
+@assets.command(help="""Download assets from the remote server.""")
 @click.option(
     "--version",
     type=str,
-    default=None,
-    help="Shinylive version to download or remove.",
+    default=_version.SHINYLIVE_ASSETS_VERSION,
+    help="Shinylive version to download.",
     show_default=True,
 )
+@click.option(
+    "--dir",
+    type=str,
+    default=None,
+    help="Directory to store shinylive assets (if not using the default).",
+)
 @click.option(
     "--url",
     type=str,
     default=None,
     help="URL to download from. If used, this will override --version.",
+)
+def download(
+    version: str,
+    dir: Optional[str | Path],
+    url: Optional[str],
+) -> None:
+    if version is None:  # pyright: ignore[reportUnnecessaryComparison]
+        version = _version.SHINYLIVE_ASSETS_VERSION
+    _assets.download_shinylive(destdir=upgrade_dir(dir), version=version, url=url)
+
+
+@assets.command(
+    help=f"Remove all versions of local assets except the currently-used version, {_assets.SHINYLIVE_ASSETS_VERSION}."
```

Comment:
When running `shinylive assets`, it prints this help:

```
  cleanup             Remove all versions of local assets except the...
```

Which has the help info truncated. But then when the user goes to run `shinylive assets cleanup`, it actually does the thing instead of printing out the full help.

Can this be made to be not truncated?

Reply:
Yes! `short_help` is not truncated.